pages/new-mission.vue

```diff
@@ -0,0 +1,486 @@
+                aria-describedby="newMissionType-live-feedback"
+              />
+
+              <b-form-invalid-feedback id="newMissionType-live-feedback" />
+            </b-form-group>
+          </b-col>
+        </b-row>
+
+        <b-row>
+          <b-col>
+            <b-form-group id="durationGroup">
+              <b-form-input
+                id="newMissionDuration"
+                v-model="$v.form.duration.$model"
+                placeholder="duration"
+                name="newMissionDuration"
+                type="text"
+                :state="$v.form.duration.$dirty ? !$v.form.duration.$error : null"
+                aria-describedby="newMissionDuration-live-feedback"
+              />
+
+              <b-form-invalid-feedback id="newMissionDuration-live-feedback" />
+            </b-form-group>
+          </b-col>
+
+          <b-col>
+            <b-form-group id="TeamSizeGroup">
+              <b-form-input
+                id="newMissionTeamSize"
+                v-model="$v.form.teamSize.$model"
+                placeholder="teamSize"
+                name="newMissionTeamSize"
+                type="number"
+                :state="$v.form.teamSize.$dirty ? !$v.form.teamSize.$error : null"
+                aria-describedby="newMissionTeamSize-live-feedback"
+              />
+
+              <b-form-invalid-feedback id="newMissionTeamSize-live-feedback" />
+            </b-form-group>
+          </b-col>
+
+          <b-col>
+            <b-form-group id="locationGroup">
+              <b-form-input
+                id="newMissionLocation"
+                v-model="$v.form.location.$model"
+                placeholder="location"
+                name="newMissionLocation"
+                type="text"
+                :state="$v.form.location.$dirty ? !$v.form.location.$error : null"
+                aria-describedby="newMissionLocation-live-feedback"
+              />
+
+              <b-form-invalid-feedback id="newMissionLocation-live-feedback" />
+            </b-form-group>
+          </b-col>
+        </b-row>
+
+        <b-row>
+          <b-col>
+            <v-select v-model="list"
+                      label="name"
+                      taggable
+                      multiple
+                      :options="environments"
+                      @input="addToDrop"
+            />
+          </b-col>
+
+          <b-col>
+            <draggable
+              tag="transition-group"
+              :component-data="componentData"
+              :list="list"
+              class="list-group"
+              draggable=".item"
+              :animation="100"
+              @start="dragging = true"
+              @end="dragging = false"
+            >
+              <div
+                v-for="element in list"
+                :key="element.name"
+                class="list-group-item item"
+              >
+                <b-row style="align-items: center">
```

Comment:
I don't think inline style is useful here, we could create a css file to regroup all the css and avoid modification on two files